metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -99,32 +122,40 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
                     self.config.filter,
                     serverctrls=[self.lc],
                 )
-                rtype, rdata, rmsgid, serverctrls = self.ldap_client.result3(msgid)
+                _rtype, rdata, _rmsgid, serverctrls = self.ldap_client.result3(
+                    msgid
+                )
             except ldap.LDAPError as e:
                 self.report.report_failure(
                     "ldap-control", "LDAP search failed: {}".format(e)
                 )
                 break
 
             for dn, attrs in rdata:
-                # TODO: create groups if 'organizationalUnit' in attrs['objectClass']
-
                 if (
                     b"inetOrgPerson" in attrs["objectClass"]
                     or b"posixAccount" in attrs["objectClass"]
                 ):
                     yield from self.handle_user(dn, attrs)
 
+                if (
+                    b"posixGroup" in attrs["objectClass"]
+                    or b"organizationalUnit" in attrs["objectClass"]
```

Comment:
I did not get enough relevant hits using only organizationalUnit, but including posixGroup might be too heavy for some use-cases. Should this be configurable?

Reply:
Yep I think making this configurable would be good, but it's also probably fine to have this enabled by default